### PR TITLE
NEWTAG-478

### PR DESCRIPTION
**Pull request by thpr**
Deprecate CHOOSE:NUMBER, replace with TEMPVALUE:

### CODE REVIEW

**Comment by LegacyKing** [comment]
Other than the deprecated Langauto and CHOOSE:LANGAUTO files tied into the PR, the code "looks" clean. (IS NOT A JAVA MONKEY)

File: code/src/java/plugin/lsttokens/TempValueLst.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		if (tok.countTokens() != 3)
+		{
+			return new ParseResult.Fail(
+				getTokenName()
+					+ " requires three arguments, MIN=, MAX= and TITLE= : "
+					+ value, context);
+		}
+		if (!tok.nextToken().startsWith("MIN="))
+		{
+			return new ParseResult.Fail("COUNT:" + getTokenName()
```
Comment (jdempsey-au) [inline code/src/java/plugin/lsttokens/TempValueLst.java]: Should that COUNT: prefix be in these messages? It seems out of place.

Comment (thpr) [inline code/src/java/plugin/lsttokens/TempValueLst.java]: Corrected, thanks for the catch :)

**Comment by jdempsey-au** [comment]
Tom, are you able to resolve the merge conflicts please?

**Comment by thpr** [comment]
Merged

**Comment by LegacyKing** [comment]
@thpr this PR still shows as conflicted.

**Comment by thpr** [comment]
Closing due to data conflicts

**Pull request closed without merging**